mixserv/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,7 @@
+# Set everything to be logged to the console
```

Comment:
Im not familiar with the plugin though, can it put `core/src/main/resources/log4j.properties` into hivemall-mixserv-xxx.jar by modifying pom.xml?